### PR TITLE
Fund diligence

### PR DESCRIPTION
**Pull request by erin-koen**
Aggregate all diligence info into one component

### CODE REVIEW

File: src/components/Routes/Fund/FundDiligence/FundContracts/FundContracts.tsx [hunk 1]
```diff
@@ -34,28 +40,33 @@ export const FundContracts: React.FC<FundContractsProps> = ({ address }) => {
...
               </DictionaryData>
             </DictionaryEntry>
-            {(a.name === 'Fund' || a.name === 'Vault') && (
+            {/* {(a.name === 'Fund' || a.name === 'Vault') && (
               <DictionaryEntry>
                 <DictionaryLabel>&nbsp;</DictionaryLabel>
                 <DictionaryData>&nbsp;</DictionaryData>
               </DictionaryEntry>
-            )}
+            )} */}
```
Comment (iherger) [inline src/components/Routes/Fund/FundDiligence/FundContracts/FundContracts.tsx:67]: Remove unused code.

File: src/components/Routes/Fund/FundDiligence/FundFinancials/FundFinancials.tsx [hunk 2]
```diff
@@ -262,57 +189,17 @@ export const FundFactSheet: React.FC<FundFactSheetProps> = ({ address }) => {
...
+{
+  /* <DictionaryEntry>
+<DictionaryLabel>Inception</DictionaryLabel>
+<DictionaryData>{creation ? <FormattedDate timestamp={creation} /> : 'N/A'}</DictionaryData>
+</DictionaryEntry>
+<DictionaryEntry>
+<DictionaryLabel>Status</DictionaryLabel>
+<DictionaryData>{fund.isShutDown ? 'Inactive' : 'Active'}</DictionaryData>
+</DictionaryEntry> */
+}
```
Comment (iherger) [inline src/components/Routes/Fund/FundDiligence/FundFinancials/FundFinancials.tsx:205]: Remove unused code.

Comment (erin-koen) [reply src/components/Routes/Fund/FundDiligence/FundFinancials/FundFinancials.tsx]: done

File: src/components/Routes/Fund/FundDiligence/FundContracts/FundContracts.tsx (hunk 1 above)
Comment (erin-koen) [reply src/components/Routes/Fund/FundDiligence/FundContracts/FundContracts.tsx]: done

**Pull request merged**